runtime/criusupport/criusupport.cpp

```diff
@@ -543,9 +544,13 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		}
 
 		malloc_trim(0);
+		Trc_CRIU_before_checkpoint(currentThread, j9time_nano_time(), j9time_current_time_nanos(&success));
```

Comment:
will the trace points work without syslog?

Reply:
The trace point actual works w/o the syslog.
`-Xtrace:trigger=tpnid{j9criu.7-12,javadump},print={j9criu}`
```
Performing CRIUSupport.checkpointJVM(), current thread name: main, Fri Oct 28 12:40:48 PDT 2022, System.currentTimeMillis(): 1666986048323, System.nanoTime(): 83919306062082
19:40:48.416*0x1aa00          j9criu.9        > Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl
JVMDUMP034I User requested Java dump using '/root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124048.29608.0001.txt' through -Xtrace:trigger
JVMDUMP010I Java dump written to /root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124048.29608.0001.txt
19:40:48.461 0x1aa00          j9criu.13       - Before checkpoint, checkpointNanoTimeMonotonic = 83919443688418, checkpointNanoUTCTime = 1666986048461130947
19:40:48.779 0x1aa00          j9criu.12       - Current syslogOptions: error,vital
JVMDUMP034I User requested Java dump using '/root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124048.29608.0002.txt' through -Xtrace:trigger
JVMDUMP010I Java dump written to /root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124048.29608.0002.txt
19:40:48.808 0x1aa00          j9criu.7        - Before checkpoint criu_dump(), j9time_nano_time() returns 83919791275562, j9time_current_time_nanos() returns 1666986048808717710
JVMDUMP034I User requested Java dump using '/root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124048.29608.0003.txt' through -Xtrace:trigger
JVMDUMP010I Java dump written to /root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124048.29608.0003.txt
Killed
```

```
19:45:05.876 0x1aa00          j9criu.8        - After checkpoint criu_dump(), j9time_nano_time() returns 84176858652809, j9time_current_time_nanos() returns 1666986305876095408
JVMDUMP034I User requested Java dump using '/root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124505.29608.0004.txt' through -Xtrace:trigger
JVMDUMP010I Java dump written to /root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124505.29608.0004.txt
19:45:05.903 0x1aa00          j9criu.14       - After restore, restoreNanoUTCTime = 1666986305876095408, checkpointNanoUTCTime = 1666986048461130947, checkpointRestoreTimeDelta = 257414964461, restoreNanoTimeMonotonic = 84176858652809, checkpointNanoTimeMonotonic = 83919443688418, nanoTimeMonotonicClockDelta = 257414964391
19:45:05.906 0x1aa00          j9criu.10       < Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl
JVMDUMP034I User requested Java dump using '/root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124505.29608.0005.txt' through -Xtrace:trigger
JVMDUMP010I Java dump written to /root/ivm/tests/jdk11-1028/aqa-tests/TKG/javacore.20221028.124505.29608.0005.txt
```

Commenting out syslog close/open does cause checkpoint failure.
It appears JVM trace or javacore files don't affect checkpoint.

P.S. appended new trace points at end of the file instead. Modified the existing tracepoint since CRIU hasn't been released yet.